pandas/compat/__init__.py

```diff
@@ -123,7 +123,7 @@ def _get_lzma_file(lzma):
 
     Returns
     -------
-    class
+    lzma.LZMAFile
```

Comment:
possibly `type(LZMAFile)`?  usually this would indicate that we're getting an instance of that class

Reply:
Should annotate the function return here as well

Reply:
> Should annotate the function return here as well

I thought of that. but the problem is that the function tests for the imports of the ```lzma``` module. If I import it beforehand it kinda losing it's point (I think)

Reply:
> Should annotate the function return here as well

@WillAyd I tried it now, I imported the ```lzma``` module at the beginning of the file and annotated ```_get_lzma_file``` with ```-> lzma.LZMAFile```, and ```mypy``` complained:

```
pandas/io/common.py:468: error: "LZMAFile" not callable
``` 

Any thoughts?